archlinuxcn/mesen/lilac.yaml

```
@@ -0,0 +1,12 @@
+maintainers:
+  - github: meanjollies
```

```suggestion
  - github: CrankySupertoon
```

You are the maintainer, not someone else. It has nothing to do with who the maintainer on AUR is.